### DISCUSSION TITLE
Always get the focus on the master database after lock

### DISCUSSION THREAD

**Discussion by T4ngml** [keepassxreboot/keepassxc#9156] [Q&A] [answered]
I'm using the AutoOpen functionnality to unlock all my databases at once. 

After a few minutes, I get locked out based on the Security > Lock databases after inactivity of..._ setting. So that I need to unlock my databases again.

I realized that I could unlock all my databases at once with the master db password, only if the focus is on the master database.
Otherwise, if I left KeePass, with focus on another database, next time I'll open KeePassXC, I'll be prompted for that specific database credentials, which won't unlock the other ones.
If I want to unlock all of them at once, I have to move the focus back to the master db thumbnail, and unlock that one.

I feel like it would be more user-friendly to get the focus back automatically to the master db. 
I understood this could be achieved using KeePass triggers. Since KeePassXC won't give access to triggers, I was wondering if there's some built-in functionality for this?

Thanks

**Comment by droidmonkey** [comment, marked as answer]
Change your keepassxc shortcut to add your main database file as a command line parameter. That will select your main database tab when you open the program.

**Comment by T4ngml** [comment]
Thanks, but by 'open' I meant when I didn't manually closed KeePassXC . I mean when the software is still open but the databases are locked after a while. Is there a way to move the focus automatically to a specific database?

**Reply by droidmonkey** [reply]
No and that's not something we would develop. That is extremely specific to your use case. You can always press Alt + 1 to choose the first database in the tab list.
